app/views/pages/moving-guide/civilians.html.erb

```diff
@@ -1,11 +1,331 @@
+      <p>The moving company will be creating an inventory of all your household items and will put stickers on all the boxes and larger items (couches, large appliances, etc.). On these inventory sheets, they will also be indicating whether or not there was any pre-existing damage to your household goods. If you disagree with their assessment, make sure you annotate that on the inventory sheet!</p>
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tips:</strong>
+        <ul>
+          <li>Do a walkthrough with the driver to make sure everything is out of your residence and loaded up. This will ensure that nothing gets left behind and if you see any damage to the residence you can annotate that on the forms provided.</li>
+          <li>Ask the driver for their phone number so you can reach them if needed.</li>
+          <li>If you don’t agree with something on that inventory form make sure you annotate that BEFORE signing in the remarks section!</li>
+        </ul>
+      <% end %>
+    </div>
+
+    <div id="getting-there" class="article-section">
+      <h4>Getting There</h4>
+      <p>When traveling <%= abbr_tag('conus') %> to your new location, you’re authorized travel by one primary vehicle, but two if you’re relocating with dependents. You may be required to do a Privately Owned Vehicle (POV) Risk Assessment before you start your en route travel. Currently, the mileage rate for <%= abbr_tag('pov') %> reimbursement is 17 cents per mile, and tolls and any parking fees are also reimbursable. During your travel you'll be given <%= link_to 'per diem', 'https://www.defensetravel.dod.mil/site/perdiemCalc.cfm' %>. Contact your command for the number of days you're authorized for travel. </p>
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> If you're traveling with pets, you can check out pet-friendly hotels on the Web at <%= link_to 'www.petswelcome.com', 'https://www.petswelcome.com/' %>.
+      <% end %>
+
+      <p>If you’re moving <%= abbr_tag('oconus') %>, make sure everyone in your family has a valid passport unless you’re headed to Hawaii or Alaska. Family members not listed on the Orders may need visas. When you arrive, expect a few in-processing steps. Depending on your host country, this could include briefings at different offices, and trips to different bases, immediately upon arrival. <%= link_to 'Transportation Office Locator', offices_path %>
+
+      <p>During your travel keep any receipts of your out of pocket expenses,  things like hotel receipts, rental car, gasoline, tolls, etc. that may be needed when you file your voucher at the end. Visit the <%= link_to 'Reimbursements & Vouchers', page_path('moving-guide/reimbursements') %> page for details.</p>
```

Comment:
A few small suggestions:

1. bold the "keep any receipts"
1. maybe say "any and all receipts" or just "all receipts"
1. remove the double-space after the first comma